server/src/callbacks.js

```diff
@@ -15,6 +15,7 @@ Empirica.onGameStart(function ({ game }) {
 
   const round = game.addRound({
     name: "Discussion",
+    topic: game.treatment.topic
```

Comment:
This is where your problem with displaying the URL is coming from. If the treatment file has the URL stored in the `topic` attribute, then this will set the topic to the URL. But, in line 25, there is code that will look for the URL in a `treatment.url` attribute (and fail, because there isn't one) and try and download the topic from `undefined`. What we should do is set the topic to an empty string in line 18, and then have line 25 get the URL from the right treatment variable.